src/exportable/ExportableConceptRule.ts

```diff
@@ -8,7 +8,7 @@ export class ExportableConceptRule extends fshrules.ConceptRule implements Expor
   }
 
   toFSH(): string {
-    let line = `* #${this.code}`;
+    let line = `* #${this.code.includes(' ') ? `"${this.code}"` : this.code}`;
```

Comment:
This works for normal space, but should we also account for tabs or other whitespace characters?  I think maybe we should -- in which case I'd check for a match on `/\s/`.

Reply:
That's a good idea! I was going off of the [FSHCode's toString](https://github.com/FHIR/sushi/blob/master/src/fshtypes/FshCode.ts#L10) in SUSHI, but I am now realizing that change was also me...

Reply:
Ha.  Maybe.  Chances are I probably approved that other one. ;-)

Reply:
It would probably be pretty bad for a code to have a tab, but you never know...

Reply:
4edae7d tests for `/\s/` when checking to use `" "`. I haven't yet made the equivalent PR for SUSHI, but I can if you'd like!